hardhat/contracts/GaslessVoting.sol

```diff
@@ -62,7 +62,7 @@ contract GaslessVoting is IERC165, IGaslessVoter
     bytes32 private constant EIP712_DOMAIN_TYPEHASH = keccak256("EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)");
     string private constant VOTINGREQUEST_TYPE = "VotingRequest(address voter,address dao,bytes32 proposalId,uint256 choiceId)";
     bytes32 private constant VOTINGREQUEST_TYPEHASH = keccak256(bytes(VOTINGREQUEST_TYPE));
-    bytes32 private immutable DOMAIN_SEPARATOR;
+    bytes32 private DOMAIN_SEPARATOR;
```

Comment:
This one could probably stay immutable? But maybe with a comment/note saying that this exposes the value via code